pages/osx/colima.md

```diff
@@ -0,0 +1,36 @@
+# colima
+
+> Container runtimes for macOS and Linux with minimal setup.
+> More information: <https://github.com/abiosoft/colima>.
+
+- Start Colima:
+
+`colima start`
+
+- Use a config file:
+
+`colima start --edit`
+
+- Start and setup Containerd. Install nerdctl to use Containerd via nerdctl:
```

Comment:
```suggestion
- Start and setup containerd (install `nerdctl` to use containerd via `nerdctl`):
```
containerd's website writes its name in all downcase characters.
@kbdharun @tricantivu I think we shouldn't enclose "containerd" with backticks, right? From the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#descriptions):
Brand and project names can be capitalized in the description whenever applicable (e.g. use A tool for interacting with a Git repository. instead of A tool for interacting with a `git` repository.).

Reply:
Yeah, we can leave it as is is here.